pygfx/utils/text/_shaper.py

```diff
@@ -104,16 +152,8 @@ def shape_text_hb(text, font_filename, direction=None):
         buf.direction = direction
         is_horizontal = direction in ("ltr", "rtl")
 
-    # Load font
-    cached = CACHE_HB.get(font_filename)
-    if cached:
-        blob, face, font = cached
-    else:  # Cache miss
-        blob = uharfbuzz.Blob.from_file_path(font_filename)
-        face = uharfbuzz.Face(blob)
-        font = uharfbuzz.Font(face)
-        font.scale = ref_size, ref_size
-        CACHE_HB.set(font_filename, (blob, face, font))
+    # Load font, maybe from the cache
+    blob, face, font = CACHE_HB[font_filename]
```

Comment:
Generally, I really like the `lru_cache` from python's STL, but it doesn't give enough tools for inspection of the cache and tuning.

So I've always resorted to building my own custom ones that I could invalidate based on other logic (such as time in this case).

Reply:
One aspect of the lru_cache is that you don't think about the cache getter and setter, it just "does it for you".

Reply:
Yeah, this is nice!